modules-js/react-fleet/src/sectioning-elements/SectionHeader.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+export type Props = {
+  title: string;
+  yellow?: boolean;
+  subheader?: boolean;
+  className?: string;
+};
+
+export default function SectionHeader(props: Props): JSX.Element {
+  const { title, yellow, subheader } = props;
+  const classes = [
+    'sh',
+    'm-b300'
+  ];
+
+  if (yellow!) {
```

Comment:
`!` I think shouldn’t be necessary here; I would expect `undefined` is a valid statement in an `if`.